test/rsa_complex.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Check to see if there is a conflict between complex.h and openssl/rsa.h.
+ * the former defines "I" as a macro and earlier versions of the latter use
+ * for funtion arguments.
```

Comment:
nit: typo

Reply:
Thanks, fixed this and the capitalisation problem.